packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -232,7 +316,23 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
         {(type, item) => {
           if (type === 'item') {
             return (
-              <ListViewItem item={item} isEmphasized dragHooks={dragHooks} hasActions={onAction}  />
+              <>
+                {isListDroppable && state.collection.getKeyBefore(item.key) == null &&
+                  <RootDropIndicator key="root" />
+                }
+                {isListDroppable &&
```

Comment:
We should probably try to only render these extra DOM nodes when a drag session is actually in progress. Right now they render all the time. The DroppableGrid story might do something like that.